Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3105,6 +3105,12 @@ void MediaPlayerPrivateGStreamer::createGSTPlayBin(const URL& url)
 
     g_object_set(m_pipeline.get(), "mute", static_cast<gboolean>(player->muted()), nullptr);
 
+    // From GStreamer 1.22.0, uridecodebin3 is created in playbin3's _init(), so "element-setup" isn't called with it.
```

Comment:
That's imho a regression. Can you send a MR fixing this in GStreamer?

Reply:
Indeed. I will try :)

Reply:
Well, in practical/technical terms it's a regression, but maybe looking at the bigger picture it's actually an improvement. I just wish that was documented as a behaviour change in the release notes. On second thoughts maybe not really worth a MR in GStreamer. Don't worry about it.